Sources/Purchasing/Purchases/Purchases.swift

```diff
@@ -1548,7 +1548,7 @@ public extension Purchases {
     /**
      * Deprecated
      */
-    @available(*, deprecated, message: "Configure behavior through the RevenueCat dashboard instead")
+    @available(*, deprecated, message: "Configure behavior through the RevenueCat dashboard instead. If you are using the legacy restore behavior and are currently setting this to true, continue doing so.")
```

Comment:
```suggestion
    @available(*, deprecated, message: "Configure behavior through the RevenueCat dashboard instead. If have configured the "Legacy" restore behavior in the [RevenueCat Dashboard](app.revenuecat.com) and are currently setting this to `true`, keep this setting active.")
```

Reply:
we might wanna open up equivalents for other SDKs

Reply:
> we might wanna open up equivalents for other SDKs
Yeah, we should! I'll open up some PRs for it tomorrow. 👍